interface-definitions/firewall.xml.in

```diff
@@ -66,6 +66,22 @@
         </properties>
         <defaultValue>disable</defaultValue>
       </leafNode>
+      <node name="fastpath">
+        <properties>
+          <help>Software fastpath with nftables flowtable</help>
+        </properties>
+        <children>
+          <leafNode name="interface">
+            <properties>
+              <help>Interfaces to enable</help>
+              <completionHelp>
+                <script>${vyos_completion_dir}/list_interfaces --type ethernet</script>
```

Comment:
Why Ethernet only?
It could be tunnel, wireguard or PPPoE, etc.

Reply:
I was lazy at when writing this PR XD.
The reason is that when adding a virtual interface (e.g. a bridge, pppoe interface or vlan vif) to a flowtable, it actually adds the underlying physical interface. Therefore all ingress traffic from that physical interface will be offload.
e.g. if you add `eth0.3` and `eth.1` to a flowtable, forwarding traffic between `eth0.4` and `eth.1` will be offloaded as well.
So limiting it to ethernet could be less confusing for end-users.